cmd/client/help_message.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+)
+
+func restart_message() {
+	var message strings.Builder
+	message.WriteString("\033[0;31mError\033[0m: restart requires a process name\n")
```

Comment:
`\x1b` is better for compatibility. 
Also you don't need to tell 0;31 when you add a color, just say 31